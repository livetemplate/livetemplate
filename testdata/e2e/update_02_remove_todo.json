{"1":"Task Manager","2":"8","9":"2","10":"1","11":"1","13":"50","31":"2023-01-01 10:30:00","32":"session-12345"}