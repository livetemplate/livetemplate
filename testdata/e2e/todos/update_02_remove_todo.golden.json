{
  "1": "8",
  "2": "active",
  "3": "High Activity",
  "4": "2",
  "5": "0",
  "7": "0%",
  "8": "\u003cdiv class=\"todo-list\"\u003e\n                    \n                    \u003cdiv class=\"todo-item \" data-key=\"todo-1\"\u003e\n                        \u003cstrong\u003e#0:\u003c/strong\u003e\n                        Learn Go templates\n                        ○\n                         (Priority: High)\n                    \u003c/div\u003e\n                    \n                    \u003cdiv class=\"todo-item \" data-key=\"todo-3\"\u003e\n                        \u003cstrong\u003e#1:\u003c/strong\u003e\n                        Write documentation\n                        ○\n                         (Priority: Low)\n                    \u003c/div\u003e\n                    \n                \u003c/div\u003e",
  "9": "2023-01-01 10:30:00"
}